client/src/test/java/io/pravega/client/segment/impl/SegmentOutputStreamTest.java

```diff
@@ -115,28 +115,6 @@ public void testConnectAndConnectionDrop() throws Exception {
     }
 
     @Test(timeout = 10000)
-    public void testNettyConnectionFailure() throws Exception {
```

Comment:
This test case still holds with the recent changes, no?

Reply:
yes, I have reverted the changes.